tequilapi/contract/node.go

```diff
@@ -125,3 +125,12 @@ type ProviderSession struct {
 type LatestReleaseResponse struct {
 	Version string `json:"version"`
 }
+
+// EarningsPerServiceResponse contains information about earnings per service
+// swagger:model EarningsPerServiceResponse
+type EarningsPerServiceResponse struct {
+	EarningsPublic   string `json:"public"`
```

Comment:
Please use `Token` contract, as it will make string values more meaningful and it is our default contract for crypto